temporalcli/commandsgen/commands.yml

```
@@ -647,6 +682,119 @@ commands:
         description: Reason for terminating the batch job.
         required: true
 
+  - name: temporal config
+    summary: Manage config files (EXPERIMENTAL)
+    description: |
+      Config files are TOML files that contain profiles, with each profile
+      containing configuration for connecting to Temporal. 
+
+      ```
+      temporal config set \
+          --prop address \
+          --value us-west-2.aws.api.temporal.io:7233
+      ```
+
+      The default config file is at `$CONFIG_PATH/temporal/temporal.toml` where
```

```suggestion
      The default config file path is `$CONFIG_PATH/temporal/temporal.toml` where
```

Since that file may not exist until a user makes it (unless we auto-write an empty one or something on first go, but I didn't see that)

---

We definitely don't auto-write any files. Will update text.